api/models.py

```diff
@@ -369,6 +369,12 @@ class Hierarchy(BaseModel):
 Hierarchy.update_forward_refs()
 
 
+class RegressionData(BaseModel):
+    """Model for the data field of a Regression"""
+    testrun_failed: PyObjectId
+    testrun_passed: PyObjectId
```

Comment:
Field names are not aligned with https://github.com/kernelci/kernelci-pipeline/pull/343/

Reply:
The reason for having embedded models here rather than just an object ID was to reduce DB find operations for further processing.